plugins/chat/lib/chat/outgoing_web_hook_extension.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Chat
+  module OutgoingWebHookExtension
```

Comment:
I added `outgoing` as a prefix because chat already has incoming webhooks and I tried to reduce the confusion